packages/carbon-graphs/src/js/controls/PairedResult/PairedResult.js

```diff
@@ -78,6 +78,26 @@ const loadInput = (inputJSON) => new PairedResultConfig()
   .clone()
   .getConfig();
 
+/**
+ * Filters invalid data like null and blank.
+ *
+ * @private
+ * @param {object} data - Data points object
+ * @returns {object} filteredData - filtered data object
+ */
+const filterPairedResultData = (data) => {
+  let filteredData = [];
+  data.map((value) => {
+    let filteredValue = {};
+    iterateOnPairType((t) => {
```

Comment:
1. There lot of places where we iterate those constants(high, mid and low) as shown in the below image, because of this reason we are having a separate function (**[iterateOnPairType](https://github.com/cerner/terra-graphs/blob/fix-issue-11/packages/carbon-graphs/src/js/controls/PairedResult/helpers/helpers.js#L66)**) which iterates over constants.

![image](https://user-images.githubusercontent.com/63670637/100651053-7b85fc00-330a-11eb-82c4-7a0d0161c894.png)

2. changed the code accordingly. changes made in [here](https://github.com/cerner/terra-graphs/pull/19/commits/714401d7bb5ad612f2150279c73c00570f5f56e8#diff-1a3e9e5e1951c8c06a171883626624b21dfb2e6131ec69031c9cf36f5e341f36L90).

3. As per our [documentaiton](https://github.com/cerner/terra-graphs/blob/main/packages/terra-graphs/docs/controls/PairedResult.md) data should be in the form of object, so I handled when high/low/mid when {} instead of "".